build.gradle

```diff
@@ -28,9 +28,7 @@ ext {
 }
 
 dependencies {
-    // by using the compile time classpath we add the parser classes to the jar but the dependency doesn't end up
-    // in the POM's <dependency> block
-    compileOnly group: 'ch.poole', name: 'OpeningHoursParser', version: parserVersion
+    implementation group: 'ch.poole', name: 'OpeningHoursParser', version: '0.23.2'
```

Comment:
Ok, is this on Maven Central now? If yes, that's great!

Reply:
I've also updated the readme to remove the relevant part.